.env.template

```diff
@@ -20,3 +20,7 @@ NEXT_PUBLIC_SWELL_PUBLIC_KEY=
 
 NEXT_PUBLIC_SALEOR_API_URL=
 NEXT_PUBLIC_SALEOR_CHANNEL=
+
+COMMERCELAYER_CLIENT_ID=
```

Comment:
Variables here are not prefixed with NEXT_PUBLIC but the ones being used are, if those variables are being consumed in client side, they should be prefixed

Reply:
Got it! I'll fix this.